docs/advanced-usage/clerk-idp.mdx

```diff
@@ -1,115 +1,147 @@
-  ```
+### Configure your client
 
-  To create an OAuth Application that uses the authorization code flow with proof of key exchange (PKCE), set `public` to `true` in the POST request body.
+Now that you have set up a Clerk OAuth application, you will need to configure any of the following settings needed in your client.
 
-  ```json
-  {
-    "callback_url": "https://oauth-public-client.com/oauth2/callback",
-    "name": "oauth_app_pkce",
-    "public": true
-  }
-  ```
-
-  Clerk creates an OAuth application and returns it's configurations back to you.
-
-  ```json {{ filename: 'RESPONSE' }}
-  {
-    "object": "oauth_application",
-    "id": "oa_2O4BCh3zUONvWlZHtBXv6s6tm7u",
-    "instance_id": "ins_2O4Ak02T4fDmKKv6tK5h0WJ8ou8",
-    "name": "oauth_app_1",
-    "client_id": "d9g4CT4WYiCBm7EU",
-    "client_secret": "VVgbT7i6sPo7sTljq2zj12fjmg0jPL5k",
-    "scopes": "profile email",
-    "callback_url": "https://oauth-client.com/oauth2/callback",
-    "authorize_url": "https://clerk.your-domain.com/oauth/authorize",
-    "token_fetch_url": "https://clerk.your-domain.com/oauth/token",
-    "user_info_url": "https://clerk.your-domain.com/oauth/userinfo",
-    "created_at": 1680809847940,
-    "updated_at": 1680810135145
+- **Client ID**: Public identifier of your Clerk OAuth application.
+- **Client Secret**: Confidential secret used to authenticate your Clerk OAuth application.
+- **Authorize URL**: Used by the client to request authorization from your user.
+- **Token URL**: Used by the client to exchange an authorization code for an access token and a refresh token.
+- **User Info URL**: Used by the client to retrieve additional user data upon authentication.
+- **Discovery URL**: Used by the client to retrieve the configuration data of the Clerk OAuth application.
+
+### OAuth 2.0 Scopes
+
+OAuth 2.0 scopes define the level of access and specific user data that will be shared with the client application during authentication. The following scopes are currently supported:
+
+| Scope | Access |
+| - | - |
+| `profile` | Grant access to the user personal information, such as first and last name, avatar and username |
+| `email` | Grant access to the user email address |
+| `public_metadata` | Grant access to the user public and unsafe metadata |
+| `private_metadata` | Grant access to the user private metadata |
+
+### Get user information
+
+After a user has successfully completed an OAuth 2.0 flow, you can use Clerk's access token to retrieve additional user information from Clerk's [`/oauth/userinfo`](/docs/reference/frontend-api/tag/OAuth2-Identify-Provider#operation/getOAuthUserInfo){{ target: '_blank' }} endpoint, based on the granted scopes.
+
+To retrieve the OAuth access token for a user, use the [`getUserOauthAccessToken()`](/docs/references/backend/user/get-user-oauth-access-token) method from the JavaScript Backend SDK. This method must be used in a server environment, and cannot be run on the client.
```

Comment:
We should drop this paragraph as well as the example. In the case of Clerk as an OAuth 2.0 provider, the customer won't use a Clerk SDK to retrieve additional user information, but rather a package/library of their choice, which supports the OAuth 2.0 flow operations